compiler/src/typed/translprim.re

```diff
@@ -4,7 +4,6 @@ open Typedtree;
 open Parsetree;
 
 type primitive_constant =
```

Comment:
/Just thinking about this...aren't these basically `prim-1`s? 🙃

Reply:
Uh, I think you meant prim0? And yes, they're similar but not quite the same thing.